src/interfaces/protocols-configure.ts

```diff
@@ -133,8 +134,19 @@ export class ProtocolsConfigure extends Message<ProtocolsConfigureMessage> {
         );
       }
 
-      // Validate that if `who` is not set to `anyone` then `of` is set
-      if (action.who !== undefined && ['author', 'recipient'].includes(action.who) && !action.of) {
+      // Validate that if `who === recipient` and `of === undefined`, then `can` is either `delete` or `update`
```

Comment:
Your PR comment on why only `delete` and `update` are valid is very insightful, can you add those to the comments here?

Reply:
Added